SPARQLWrapper/Wrapper.py

```diff
@@ -507,7 +507,7 @@ def _createRequest(self):
         if self.user and self.passwd:
             if self.http_auth == BASIC:
                 credentials = "%s:%s" % (self.user, self.passwd)
-                request.add_header("Authorization", "Basic %s" % base64.b64encode(credentials.encode('utf-8')))
+                request.add_header("Authorization", "Basic %s" % base64.b64encode(credentials.encode('utf-8')).decode('utf-8'))
```

Comment:
actually after `b64encode` we should probably just use `.decode('ascii')` or `six.text_type(base64.b64encode(credentials.encode('utf-8'))` as there shouldn't be any non-ascii chars in there anymore and if it should err...